code/API_definitions/Application_Endpoint_Discovery_API.yaml

```diff
@@ -44,70 +47,35 @@ info:
     * `Phone-Number` : The device's phone number in international E.164 format.
     * `Network-Access-Identifier`: A unique identifier provided by the network
     operator.
+    * `ApplicationEndpointsID`: A unique identifier for the registered application endpoint.
```

Comment:
```suggestion
In addition the following application-specific identifiers must be provided
    * `ApplicationEndpointsID`: A unique identifier for the registered application endpoint.
    * `AppId` : a unique identifier for the application
```

Reply:
^ broke the ApplicationEndpointsID out of the 'device-specific' list, and added 'AppId'

Reply:
I think `ApplicationEndpointsID` should be `ApplicationEndpointsId` (lowercase d at the end) throughout